packages/expo-dev-launcher/plugin/src/withDevLauncherAppDelegate.ts

```diff
@@ -62,10 +81,51 @@ const DEV_MENU_IOS_INIT = `
   [DevMenuManager configureWithBridge:bridge];
 #endif`;
 
-export function modifyAppDelegate(appDelegate: string, expoUpdatesVersion: string | null = null) {
-  const shouldAddUpdatesIntegration =
-    expoUpdatesVersion != null && semver.gt(expoUpdatesVersion, '0.6.0');
+const DEV_LAUNCHER_INIT_TO_REMOVE = `RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:launchOptions];
+  RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge moduleName:@"main" initialProperties:nil];
```

Comment:
Is there a way to do this without hardcoding the moduleName `main` here and on line 111?

Reply:
If you want to change it, you can always provide the init function in AppDelegate. For our purpose,  the main should be enough.